pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -58,15 +58,21 @@ func watchFile(ctx context.Context, path string, onChanged func([]byte)) error {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
-		case <-watcher.Events:
-			data, err := ioutil.ReadFile(filepath.Clean(path))
+		case e := <-watcher.Events:
+			if !(e.Name == filePath ||
```

Comment:
`e.Name` is relative path, not absolute. So I guess you should split `filePath` into `directoryPath` and `fileName` and use 1 for watching and 2 for event filtering.

Reply:
Fixed.